docs/source/api.rst

```diff
@@ -99,6 +99,11 @@ Dataset Generation (Dask-based Multi-GPU)
   .. automodule:: cuml.dask.datasets.regression
      :members:
 
+Array classes
```

Comment:
```suggestion
Array Wrappers
```

(I always like to refer to CumlArray as a wrapper so folks know we're not trying to build a competitor to existing array classes)